x-pack/legacy/plugins/apm/server/lib/helpers/setup_request.ts

```diff
@@ -47,35 +40,70 @@ export interface Setup {
   uiFiltersES: ESFilter[];
   client: ESClient;
   internalClient: ESClient;
-  config: KibanaConfig;
+  config: APMConfig;
   indices: ApmIndicesConfig;
   dynamicIndexPattern?: IIndexPattern;
 }
 
-export async function setupRequest(req: Legacy.Request): Promise<Setup> {
-  const query = (req.query as unknown) as APMRequestQuery;
-  const { server } = req;
-  const savedObjectsClient = server.savedObjects.getScopedSavedObjectsClient(
-    req
-  );
-  const config = server.config();
-  const indices = await getApmIndices({ config, savedObjectsClient });
+export type SetupWithTimeRange = Setup & {
+  start: number;
```

Comment:
`Setup` already includes timerange?

Reply:
It shouldn't - that's a merge conflict resolution gone wrong. Will fix.

Reply:
Fixed now.